frontend/pages/index.tsx

```diff
@@ -207,6 +210,24 @@ const Home: NextPage = () => {
     },
   ];
 
+  const statistics: StatisticItemProps[] = [
+    {
+      statistic: `$${stats?.totalContributionAmount}`,
+      icon: <VolunteerActivismIcon sx={statisticsIconSx} />,
+      description: 'Raised for charities',
+    },
+    {
+      statistic: String(stats?.totalRedemptionCount),
+      icon: <LocalActivityIcon sx={statisticsIconSx} />,
+      description: 'Coupons redeemed',
+    },
+    {
+      statistic: String(stats?.totalCharitiesSupported),
```

Comment:
Let's define the type of `statistic` to be `string | number | undefined` instead